README.md

```diff
@@ -26,6 +26,12 @@ A project repo will be the place where all code, documentation and issue trackin
 
 - Bugfixes and new features should include tests[1].
 
+#### Migrations
+
+- Pull Requests should not contain migrations.  If schema changes are required, it should be indicated in the name of the PR or with a label.  Then, when the PR is reviewed and ready to merge, migrations should be added in a separate commit and merged immediately.
+
+- Migrations files should not be deleted or changed once they are in the master branch. 
```

Comment:
You can squash migrations any time, just need to be sure not to delete the old ones and then django can figure out which ones to use.  However, I did use squashmigrations once before and it messed up.  There was already an issue about it in their tracker and it was over a year ago, so I imagine it's fixed now.